tools/common/lib.go

```diff
@@ -1298,6 +1298,15 @@ func Publish(c *cli.Context, cClient ccli.Client) {
 	close(receiptCh)
 }
 
+type kafkaMessageJSON struct {
+	EnqueueTime string
+	Key         string
```

Comment:
Kafka message key and Value could be binary. Is there any specific reason that they are strings here?